src/main/java/com/elytradev/concrete/resgen/concreteblockmodel.json

```diff
@@ -0,0 +1,6 @@
+{
+  "parent": "block/cube_all",
+  "textures": {
+    "all": "#ALL"
```

Comment:
Generally, variables intermixed in normal source should be `@LIKE_THIS@`. They're less likely to be misinterpreted; especially since vanilla uses hash references itself.